spec/factories/container_nodes.rb

```diff
@@ -4,4 +4,9 @@
       x.computer_system = FactoryGirl.create(:computer_system)
     end
   end
+
+  factory :kubernetes_node,
```

Comment:
These factories should derive from the parent factory....  `factory :kubernetes_node, :parent => :container_node`